src/test/ui/lint/lint-exceeding-bitshifts.rs

```diff
@@ -15,7 +15,7 @@ pub trait Foo {
 }
 
 impl<T: Foo> Foo for Vec<T> {
-    const N: i32 = T::N << 42; // FIXME this should warn
+    const N: i32 = T::N << 42; // ERROR: arithmetic operation will overflow
```

Comment:
Why did the stderr file not change when there is a new error here?

Reply:
Ah, the comment is actually a NOP. You have to make it `//~ ERROR`.

So this PR definitely did not fix the original issue (or else PR CI would have complained that this test raises an error where none is expected).

Reply:
Oh shoot, yeah. I've fixed the test case now, and yeah, I can confirm this doesn't fix the original test case.